config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   
   get 'server', to: 'server#index'
+  get 'notifications', to: 'notifications#notifications'
```

Comment:
There was already an `index` action that brings the user to the original hybrid notifications/activities page when they first log into the site. I just needed another route to hit for the completely separate page that Ross requested that is **just** notifications. I guess I could just do some conditional logic in the controller on the activities to not try to grab them if the user is trying to access just the notifications page.